frontend/53/charcounter.js

```diff
@@ -0,0 +1,20 @@
+class CharCounter {
```

Comment:
Good clean solution. Same as the other--no need to keep HTML inside the function.